pulp_container/app/registry_api.py

```diff
@@ -447,7 +447,7 @@ def put(self, request, path, pk=None):
         if upload.sha256 == digest[len("sha256:") :]:
             try:
                 artifact = Artifact(
-                    file=upload.file.name,
+                    file=upload.file.path,
```

Comment:
I changed it to path because with this change  https://github.com/pulp/pulp_container/pull/745/files#diff-1707426fbe1933a12da3490e42f0dbceea79e47f7c883708c0b0cb49755b1d94R364 file.name is no longer full path as it was before. We can afford using file.path directly here because this plugin version still uses core implementation of chunks which are stored on the local filesystem.